watertap/costing/units/anaerobic_digestor.py

```diff
@@ -0,0 +1,92 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pyomo.environ as pyo
+from ..util import (
+    register_costing_parameter_block,
+    make_capital_cost_var,
+)
+
+
+def build_anaerobic_digestor_cost_param_block(blk):
+
+    blk.capital_a_parameter = pyo.Var(
+        initialize=19.3552312e6,
+        doc="A parameter for capital cost",
+        units=pyo.units.USD_2012,
+    )
```

Comment:
Maybe leave a note here mentioning that these values came from the anaerobic_digestion_oxidation.yaml?

Reply:
Yeah, it's good to have a reference and I've added a comment.

Reply:
It'd be even better if we noted the original source rather than the yaml

Reply:
It seems like the original source of this data is the "NREL Waste-to-Energy Model". I got this info from cell A890 in the VAR tab of this Excel Kurby sent me earlier: https://app.box.com/file/1177021287137.

Reply:
Thanks, @MarcusHolly. I've updated the reference